packages/flutter/lib/src/gestures/velocity_tracker.dart

```diff
@@ -208,6 +208,34 @@ class _LeastSquaresVelocityTrackerStrategy extends _VelocityTrackerStrategy {
 
 }
 
+/// A velocity in two dimensions.
+class Velocity {
```

Comment:
Another option (rather than wrapping an Offset) would be to make Velocity inherit from OffsetBase, like Offset and Size.